go/tasks/plugins/array/k8s/task.go

```diff
@@ -46,6 +47,29 @@ const (
 	LaunchReturnState
 )
 
+const finalizer = "array.k8s.finalizers.flyte.k8s.io"
```

Comment:
can we instead call it 
```suggestion
const finalizer = "finalizers.array.flyte.org"
```

Reply:
RuntimeExecutionError: failed during plugin execution, caused by: failed to execute handle for plugin [k8s-array]: [SUBMIT_JOB_FAILED] Failed to submit job., caused by: Pod  is invalid: metadata.finalizers[0]: Invalid value: "finalizers.array.flyte.org": name is neither a standard finalizer name nor is it fully qualified

Reply:
I changed it to "flyte/array"